build.sbt

```diff
@@ -53,12 +53,13 @@ lazy val modules: List[ProjectReference] = List(
   // scalafixTests
 )
 
-lazy val root = project.in(file("."))
+lazy val root = project
+  .in(file("."))
   .enablePlugins(PrivateProjectPlugin)
   .settings(
     // Root project
     name := "http4s",
-    description := "A minimal, Scala-idiomatic library for HTTP",
+    description := "A minimal, Scala-idiomatic library for HTTP"
```

Comment:
You don't like my trailing commas? :laughing:

Reply:
I think they were removed by scalafmt; I can restore them

Reply:
I'm maintaining four branches right now, so I'm just a little bit frantic about unnecessary diffs right now, especially when they're on the furthest branch downhill.